pkg/controller/defaults/defaults.go

```diff
@@ -16,6 +16,8 @@ const (
 var (
 	// MonCount is the default number of monitors to be configured for the CephCluster
 	MonCount = 3
+	// MonCountLarge is the number of monitors to be configured for the CephCluster whenever enough nodes are available
+	MonCountLarge = 5
```

Comment:
I would recommend changing this one and above to be a more complementary pair, `MonCountMin` and `MonCountMax`.